sound/soc/sof/control.c

```diff
@@ -234,15 +232,9 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 	if (copy_from_user(&header, tlvd, sizeof(const struct snd_ctl_tlv)))
 		return -EFAULT;
 
-	/*
-	 * The maximum length that can be copied is limited by IPC max
-	 * length and topology defined length for ext bytes control.
-	 */
-	if (be->max < max_size) /* min() not used to avoid sparse warnings */
-		max_size = be->max;
-	if (header.length > max_size) {
+	if (header.length > be->max) {
```

Comment:
maybe add a note on where be->max is set, and if its value are sensible.

Reply:
be->max is coming from topology, so don't know should we have still some hard coded max value in the driver?